### PR TITLE
[enterprise-4.10]TELCODOCS-663:Add information about must-gather images for Poison Pill, NHC, and NMO

### PR DESCRIPTION
**Pull request by abhatt-rh**
[TELCODOCS): This PR adds information about must-gather images for the following Operators:

- Poison Pill
- Node Health Check
- Node Maintenance

Version(s): OpenShift 4.10 only (enterprise-4.10)

Issue:
https://issues.redhat.com/browse/TELCODOCS

Previews:

- [Gathering data about specific features](http://file.rdu.redhat.com/avbhatt/ent-TD-663/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data)
- Poison Pill: http://file.rdu.redhat.com/avbhatt/ent-TD-663/nodes/nodes/eco-poison-pill-operator.html#gather-data-poison-pill
- Node Health Check: http://file.rdu.redhat.com/avbhatt/ent-TD-663/nodes/nodes/eco-node-health-check-operator.html#gather-data-nhc
- Node Maintenance: http://file.rdu.redhat.com/avbhatt/ent-TD-663/nodes/nodes/eco-node-maintenance-operator.html#gather-data-nmo

### CODE REVIEW

**Comment by abhatt-rh** [comment]
@anna-savina @prabinovRedhat,
Please review these changes for must-gather images for the Poison Pill, NHC and NMO operators. Thank you!
cc: @slintes
The changes are same as https://github.com/openshift/openshift-docs/pull/48062 - Only that this PR is applicable for Poison Pill 4.10 and adds relevant image for NMO 4.10

**Review by tmalove — COMMENTED** [review]
LGTM

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Comment by anna-savina** [comment]
@abhatt-rh for PPv0.3.0 and NHCv0.2.0 works this must-gather image
`registry.redhat.io/openshift4/poison-pill-must-gather-rhel8:v4.10.0`

@slintes are you sure that for SNR in 4.11 we should to use version of SNR, not clusterversion?

File: modules/gathering-data-specific-features.adoc [hunk 1]
```diff
@@ -57,6 +57,12 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:1.2.0`
 |Data collection for {sandboxed-containers-first}.
 
+|`registry.redhat.io/openshift4/poison-pill-must-gather-rhel8:v0.3.0`
```
Comment (slintes) [inline modules/gathering-data-specific-features.adoc:60]: as discussed in chat, please use `v4.10`

File: modules/gathering-data-specific-features.adoc [hunk 2]
```diff
@@ -57,6 +57,12 @@ endif::openshift-dedicated[]
...
+|Data collection for the Poison Pill Operator and the Node Health Check Operator.
+
+|`registry.redhat.io/workload-availability/node-maintenance-must-gather-rhel8:v4.10.0`
```
Comment (slintes) [inline modules/gathering-data-specific-features.adoc:63]: interesting... we do NOT have a `v4.10` for this one 🤔

**Review by slintes — CHANGES_REQUESTED** [review]

File: modules/gathering-data-specific-features.adoc (hunk 1 above)
Comment (slintes) [reply modules/gathering-data-specific-features.adoc:60]: if we want to be consistent with the NMO and SNR must gather images, which don't have a `vX.Y` version but `vX.Y.Z` only, `v.4.10.1` might be a better choice, no clue... technically both are the same.

File: modules/gathering-data-specific-features.adoc (hunk 2 above)
Comment (abhatt-rh) [reply modules/gathering-data-specific-features.adoc:63]: @anna-savina, @slintes and @prabinovRedhat 
Updated the image name per our discussion. PTAL and ack. Thank you!

**Comment by anna-savina** [comment]
LGTM

**Comment by slintes** [comment]
lgtm

**Comment by prabinovRedhat** [comment]
lgtm

**Pull request merged**